cmd/overlay.go

```diff
@@ -19,16 +19,20 @@ Options:
 	--verbose/-v		enable verbose output
 	
 Commands:
-	new [directory]		overlay a directory
-	commit			commit the changes
-	discard			discard the changes
-	list			list the active overlays
+	new [directory]			Overlay a directory
+	commit [directory]		Commit the changes
+	discard [directory]		Discard the changes
+	list					List the active overlays
 
 Examples:
-	almost overlay new /etc/cute-path
-	almost overlay commit
-	almost overlay discard
-`)
+	# almost overlay new /etc/cute-path
```

Comment:
why are those commented?

Reply:
This was supposed to show that it should be run as root. I understand if you think it should not be like this

Reply:
Mh ok it makes sense but other commands follow a different syntax and could get confusing